lib/hbase/hbase-client.js

```diff
@@ -272,38 +291,24 @@ HbaseClient.prototype.getAccountPayments = function (options, callback) {
 
 HbaseClient.prototype.getAccountBalanceChanges = function (options, callback) {
   var keyBase = options.account;
-  var table = 'lu_account_balance_changes';
-  var startRow;
-  var endRow;
+  var table = 'account_balance_changes';
+  var startRow = options.account + '|' + utils.formatTime(options.start);
+  var endRow   = options.account + '|' + utils.formatTime(options.end);
 
-  if (!options.currency) callback({error:"must provide a currency", code:400});
+  if (!options.currency) { callback({error:"must provide a currency", code:400}); }
```

Comment:
currency is no longer required here.  We should have filters for currency and issuer